docs/github-release-notes.md

```diff
@@ -0,0 +1,21 @@
+# Overview
+These standards are in place to make release notes standard across the programme so that changes can be easily identified including new features and breaking changes.
```

Comment:
I think we need to be clearer why we have this standard and what the scope of it is.

ie we want GitHub releases to be consistent and these standards are for when we are manually creating GitHub releases.